srcs/cgi/Cgi.cpp

```diff
@@ -119,17 +126,41 @@ void Cgi::Execute() {
 
   ret_val = socketpair(AF_UNIX, SOCK_STREAM, 0, socket_fds);
   if (ret_val == -1) {
-    throw std::runtime_error("CGI socket error");
+    throw std::runtime_error("CGI error: socket error");

```

Comment:
I'm sorry but I think in relation to #49, we should throw ft::Socket::serverInternalError in this these cases. I'm sorry, when I wrote the original note about runtime error, I thought we would close the server in a fatal error, but that is wrong. 
Could you please modify this PR 🙏🏻